src/array.js

```diff
@@ -358,29 +362,26 @@ TransisArray.from = function(a, mapFn, thisArg) {
   //
   // Returns the receiver.
   this.prototype.proxy = function(to, name) {
-    if (this.__proxy__) { this.unproxy(); }
-
-    this.__proxy__ = {to, name};
+    TransisObject.prototype.proxy.call(this, to, name);
```

Comment:
wouldn't the lookup invoke that method?

Reply:
We have to invoke the `proxy` method from `Transis.Object`. Here `this.proxy` is the method we're in so doing that would cause unbounded recursion and blow the stack.

Reply:
ah, missed that.